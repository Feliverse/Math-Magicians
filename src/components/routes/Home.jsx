import '../styles/Home.css'

const Home = () => (
    <section className="home">
        <h2>Welcome to our page!</h2>
        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam vitae metus in lectus fringilla suscipit nec et magna. Suspendisse non sem id urna elementum ultricies sed a leo. Sed semper pharetra magna, at maximus arcu placerat ut. Ut eleifend ut libero vitae pretium. Mauris aliquam commodo molestie. Donec posuere aliquet feugiat. In eros eros, molestie vel accumsan id, faucibus ut lectus.</p>
        <p>Nam a dui lorem. Sed at odio velit. Aenean magna sem, semper quis porta nec, tincidunt non ligula. Maecenas at lobortis risus. Quisque volutpat mi id sapien sodales, in iaculis lorem tempor. Vivamus vestibulum maximus mi ac dapibus. Donec rhoncus sem ut eros efficitur feugiat. Integer eu nisl sed lectus imperdiet aliquam. Cras nec vestibulum est, eu convallis nisl.</p>
    </section>
);

export default Home;