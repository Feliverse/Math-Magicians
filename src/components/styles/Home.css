.home {
  font-family: 'Times New Roman', Times, serif;
  padding: 2em;
}
