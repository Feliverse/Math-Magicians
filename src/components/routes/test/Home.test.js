import { render } from '@testing-library/react';
import Home from '../Home';

test('renders Home component', () => {
  const { asFragment } = render(<Home />);
  expect(asFragment()).toMatchSnapshot();
});