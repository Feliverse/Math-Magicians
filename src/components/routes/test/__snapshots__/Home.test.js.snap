// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Home component 1`] = `
<DocumentFragment>
  <section
    class="home"
  >
    <h2>
      Welcome to our page!
    </h2>
    <p>
      Lorem ipsum dolor sit amet, consectetur adipiscing elit.
    </p>
    <p>
      Nam a dui lorem. Sed at odio velit. Aenean magna sem, semper quis porta nec, tincidunt.
    </p>
  </section>
</DocumentFragment>
`;
